app/src/main/java/cm/aptoide/pt/AptoideApplication.java

```diff
@@ -941,6 +942,16 @@ public ReadPostsPersistence getReadPostsPersistence() {
     return readPostsPersistence;
   }
 
+  public String getVersionCode() {
+    String version = null;
+    try {
+      PackageInfo pInfo = getPackageManager().getPackageInfo(getPackageName(), 0);
+      version = pInfo.versionName;
+    } catch (PackageManager.NameNotFoundException e) {
```

Comment:
Can we return something like "NaN" - This should never happen but that way we could easily track the exception.